src/main/java/uk/gov/hmcts/dts/fact/entity/CourtLocalAuthority.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.dts.fact.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "search_courtlocalauthorityareaoflaw")
+@Getter
+@Setter
+@NoArgsConstructor
+public class CourtLocalAuthority {
```

Comment:
Could we rename this entity class to CourtLocalAuthorityAreaOfLaw to reflect the database table name?

Reply:
changed name to reflect database table name